src/page.js

```diff
@@ -120,12 +110,35 @@ export default class Page {
     }
   }
 
+  toMatchSnapshot() {
+    this.testConfig.isJest = true;
+    this.testStats = (expect.getState && expect.getState()) || null;
+    if (this.testStats) {
+      this.testConfig.testName = this.testStats.currentTestName;
+      this.prefixedLogger = logger.prefix(this.testConfig.testName);
+      this.testConfig.testPath = this.testStats.testPath;
+      this.testConfig.isUpdate = this.testStats.snapshotState._updateSnapshot === 'all' || false;
+    } else {
+      this._logError('Failed to get Jest test status.');
+    }
+  }
+
   async _compareImage() {
     if (!this.error) {
+      let result;
       try {
-        return await compareImage(this.globalConfig, this.config.testName);
+        result = await compareImage(this.image, this.globalConfig, this.testConfig);
       } catch (error) {
         this._logError(error);
+        throw error;
+        // return false;
```

Comment:
why is this commented out?

Reply:
lol, left over again. `throw error;` should go away as well

Reply:
Done